webrender/src/renderer.rs

```diff
@@ -664,7 +658,7 @@ impl Renderer {
             profile_counters: RendererProfileCounters::new(),
             profiler: Profiler::new(),
             enable_profiler: options.enable_profiler,
-            enable_msaa: options.enable_msaa,
+            _enable_msaa: options.enable_msaa,
```

Comment:
We could possibly just remove the MSAA related code altogether, since it's unused. But it's fine to leave here for now if you like, we could do that as a follow up.